teos/src/responder.rs

```diff
@@ -290,20 +281,33 @@ impl Responder {
                     dispute_txid
                 ),
                 x => unreachable!(
-                    "`Carrrier::send_transaction` shouldn't return this variant: {:?}",
+                    "`Carrier::send_transaction` shouldn't return this variant: {:?}",
                     x
                 ),
             }
+
+            // Try to rebroadcast the penalty tx.
```

Comment:
This should be conditional on the state of the dispute. There is no point trying to send the penalty if the dispute it not, at least, in the mempool (as in already there, or just there as a result of us pushing it)

Reply:
Fine. I was thinking of it as a trigger though (being on-chain/mempool/or not).
My view is that if a penalty is decrypted then that proves that a trigger has been triggered thus it should be sent (thinking non-LN right now).